.travis.yml

```diff
@@ -2,14 +2,20 @@ language: java
 sudo: false
 jdk:
   - oraclejdk8
-
-install: mvn clean package 
```

Comment:
dedicated settings file provided since this file "imports" the environment variable which holds the secrets for GPG_KEY_NAME and GPG_PASSPHRASE into maven.
Phase changed from "package" to "verify" since code signing binds per default to "verify".